src/pawelak-arek/Fsd.Arek.Client.Ex2/quizController.js

```diff
@@ -0,0 +1,62 @@
+var app = app || {};
+
+(function () {
+    var questionData;
+    var indexQuestion = 0;
+    correct_answer = false;
+
+    var createQuiz = function (data) {
+        var count = 0;
+        questionData = data;
+        quizStatus = false;
+
+        quizView.showQuestion(questionData[indexQuestion].question);
+        quizView.showAnswer(questionData[indexQuestion].answer);
+
+        quizWindow.buttonNext.click(
+            function () {
+                if (!quizStatus) {
+                    indexQuestion++;
+                    correct_answer === true ? count++ : count;
```

Comment:
zupełnie nie rozumiem tej linijki... wyrażenie a ? b : c bez przypisania wyniku ? przecież to nic nie daje

Reply:
w funkcji checkAnswer przypisujemy wartość true lub false (w zależności od tego czy odpowiedz jest poprawna czy nie) do zmiennej corect_answer. Po kliknięciu przycisku next sprawdzamy stan zmiennej jeżeli true to count++ (count tj. zmienna od zliczenia prawidłowych odpowiedzi).